Removing styled components

laithAwad opened the pull request:
Removed an import statement for the "styled components" from Client/src/Components/DataCell/DataCell.js that is not being used and which was causing a bug in building the frontend via docker on Mac OS/Windows.

laithAwad committed:
Revert "Removing styled components"

This reverts commit d67df1bea19afec4a18884aa55153ba7ea1ee1f4.

laithAwad committed:
Removing styled components

Removed the import statement for the "styled components" that is not being used and which was causing a bug in building via docker

TiraFire reviewed (approved):
Looks good, simple change but needed!

laithAwad committed:
Removing styled components

Removed the import statement for the "styled components" that is not being used and which was causing a bug in building via docker

JeremiahTee commented:
Approved as the change was properly committed and pushed.
